### PR TITLE
Only show invite friends link if you have the experiment

### PR DESCRIPTION
**Pull request by gmethvin**

### CODE REVIEW

File: kifi-backend/modules/shoebox/public/index.html [hunk 1]
```diff
@@ -364,7 +364,7 @@ <h3 class="collection" data-template data-id="{{id}}" style="display:none">
 		<ul class="about-kifi">
 			<li><a href="blog" class="updates-features">Kifi Updates</a>
 			<li><a href="/install">Install Kifi on this computer</a>
-			<li><a href="/site/friends/invite">Invite your friends</a>
+			<li><a href="/site/friends/invite" id="invite_friends_link" style="display:none">Invite your friends</a>
```
Comment (2is10) [inline kifi-backend/modules/shoebox/public/index.html]: Should this be `href="friends/invite"` now?

Comment (2is10) [inline kifi-backend/modules/shoebox/public/index.html]: Let’s keep with this page’s custom of using hyphens in IDs and class names.

**Comment by 2is10** [comment]
Thanks for doing this.

File: kifi-backend/modules/shoebox/public/index.html (hunk 1 above)
Comment (gmethvin) [inline kifi-backend/modules/shoebox/public/index.html]: thanks for catching the `/site/`. I guess we never noticed because of the redirect.

**Pull request merged**